modules/system_monitor.py

```diff
@@ -226,9 +226,20 @@ def _extract_task_id(*args):
         return uuid.uuid4().hex
 
 
+def _get_system_monitor_config(request: gr.Request):
+    headers = dict(request.headers)
+
+    # take per-task config as priority instead of global config
+    monitor_addr = headers.get('x-diffus-system-monitor-url', '') or headers.get('X-Diffus-System-Monitor-Url', '')
+    system_monitor_api_secret = headers.get('x-diffus-system-monitor-api-secret', '') or headers.get('X-Diffus-System-Monitor-Api-Secret', '')
+    if not monitor_addr or not system_monitor_api_secret:
+        monitor_addr = modules.shared.cmd_opts.system_monitor_addr
```

Comment:
why not set `modules.shared.cmd_opts.system_monitor_addr` as the default value of `get`
```python
system_monitor_api_secret = headers.get('x-diffus-system-monitor-api-secret', modules.shared.cmd_opts.system_monitor_addr)
```